src/gui/templates/contact.html

```diff
@@ -18,4 +18,33 @@ <h2>Kontakt</h2>
           <input type="submit" class="ig-button" value="Senden">
         </form>
       </div>
+
+      <h2>Team</h2>
+      {% is_staff %}
+
+        {% for user in author %}
```

Comment:
What is `author`? I think you need to set the variable in the view first. That means that line https://github.com/digitalfabrik/ish-goalkeeper/blob/master/src/gui/views/contact.py#L16 needs a `context` being set as in https://github.com/digitalfabrik/ish-goalkeeper/blob/master/src/gui/views/contact.py#L32. The context needs to contain a list of existing users, over which you can then iterate in the template.